teacher.rb

```diff
@@ -0,0 +1,12 @@
+require_relative './person'
+
+class Teacher < Person
+  def initialize(specialization)
+    super(10, name, parent_permission: true)
```

Comment:
- [ ] You should add `age`, `name`, and `parent_permission` as parameters in `initialize then pass them in `super` 🙃 . Please kindly fix that 😉

Reply:
Okay, thanks @cynthiainga for your review. I'll add those parameters to the method.